libmscore/harmony.cpp

```diff
@@ -1340,13 +1298,58 @@ void Harmony::layout1()
 
 void Harmony::calculateBoundingRect()
       {
-      if (textList.empty())
+      if (textList.empty()) {
             TextBase::layout1();
+            // When in EDIT mode, the height of the bounding box is always lower then the
+            // height in NORMAL mode. So, to prevent layout changes, make sure the height
+            // is always the same.
+            QRectF rect = bbox().translated(0.0, _harmonyHeight - bbox().height());
+            rect.setHeight(_harmonyHeight);
+            setbbox(rect);
+            }
       else {
             QRectF bb;
-            for (const TextSegment* ts : textList)
+            for (TextSegment* ts : textList)
                   bb |= ts->tightBoundingRect().translated(ts->x, ts->y);
-            setbbox(bb);
+
+            qreal yy = ipos().y();
+            qreal hb = lineHeight() - TextBase::baseLine();
+            if (align() & Align::BOTTOM)
```

Comment:
better brace this too, as the other branches are braced too

Reply:
This is one of the places where the coding style confuses me :-)